### DISCUSSION TITLE
3.4.4 release triggering multiple alerts from Norton and Windows

### DISCUSSION THREAD

**Discussion by tpallred** [pawelsalawa/sqlitestudio#4756] [General]
### Details
Norton flagged SQLiteStudio-3.4.4-windows-x64-installer.exe on download as potentially unsafe.  I installed it on Windows 10 successfully after that, replacing the previous 3.4.3 version.  When it ran, both Norton and Windows flagged it as unsafe, from an unknown source, and with an invalid digital signature.  What's going on here?

### Steps to reproduce
Download and install and run SQLiteStudio-3.4.4-windows-x64-installer.exe

### Operating system
Windows 10

### SQLiteStudio version
3.4.4 replacing 3.4.3

**Comment by pawelsalawa** [comment]
SQLiteStudio binaries are not signed. More details on why: #3932
Your antivirus is just overprotective ;)
I don't know if and why it was not like that for version 3.4.3 (as you mentioned you had it already). Maybe the antivirus got its rules updated in the meantime, so it does not like unsigned binaries anymore?

**Reply by tpallred** [reply]
Had zero issues with download/install/run for 3.4.3, which was my first experience with SQLiteStudio.  That's why this freaked me out.  But the reasons it said it raised the flags seemed pretty mild, so I overrode its attempts to block the product.  I was thinking the whole time, "Damn, I"m going to be 'that guy'.  The guy who pushes past all of the warnings and installs massive malware on his system. Since it was a new release, I thought maybe it hadn't been packaged correctly.  Anyway, 3.4.4 runs fine so far, and I haven't noticed extra CPU cycles, drive activity, or network activity.  I will report back if there is news, but I'm going to continue as if Windows and Norton were being extra careful.  It will be interesting if I am the only one to report this.  Thanks.

**Reply by pawelsalawa** [reply]
Just so you know, when I released this 3.4.4 version, I've attempted to download all the released packages, just to test that all links are correct. What surprised me, was that Chrome warned me, that these binaries are downloaded very rarely and because of that they may be dangerous. I knew the reason why they were rarely downloaded, so I understood it instantly. I'm saying that, because maybe Norton shares some of these algorithms and maybe even the same database of "how often things are downloaded" with Google - who knows...

**Comment by tpallred** [comment]
Well, if anyone else has this happen, maybe they'll see this discussion and not be concerned.  Still no bad effects that I can see.  I always have task manager and other monitoring programs running so I can see if some unusual process starts using resources.  Thanks.
